From 928b84917139e42611ea1b71ab8b35bf20d2627b Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Mon, 8 Jun 2015 22:30:59 +0200
Subject: [PATCH] man: systemd.link - explain random MAC addresses

Two of the bits in the MAC address are set unconditioanlly, and the rest is randomized,
make this clear in the documentation (as it currently read as if it was all random).

(cherry picked from commit 2e229e0c4c29e8a827be9ffe361741cf5e9aa7af)
---
 man/systemd.link.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/man/systemd.link.xml b/man/systemd.link.xml
index 3fac760b01..d9b1879c59 100644
--- a/man/systemd.link.xml
+++ b/man/systemd.link.xml
@@ -229,7 +229,9 @@
                 <para>If the kernel is using a random MAC address,
                 nothing is done. Otherwise, a new address is randomly
                 generated each time the device appears, typically at
-                boot.</para>
+                boot. Either way the random address will have the
+                <literal>unicast</literal> and
+                <literal>locally administered</literal> bits set.</para>
               </listitem>
             </varlistentry>
           </variablelist>
